vmm_core/virt_mshv/src/lib.rs

```diff
@@ -973,7 +1000,7 @@ impl EmulatorSupport for MshvEmulationState<'_> {
             .flatten()
             .any(|range| {
                 (range.guest_pfn..range.guest_pfn + range.size).contains(&gpa)
-                    && (!write || range.flags & HV_MAP_GPA_WRITABLE == HV_MAP_GPA_WRITABLE)
+                    && (!write || range.flags & set_bits!(u8, MSHV_SET_MEM_BIT_WRITABLE) == set_bits!(u8, MSHV_SET_MEM_BIT_WRITABLE))
```

Comment:
My preference is to use the `hvdef` types where we can.

Reply:
Oh, perhaps your comment above is indicating that these bits are not actually the same.

Reply:
I am happy to use any other flag compatible with MSHV_SET_MEM_BIT_*, though. I couldn't find any in the code, but I might have missed them. We just can't use the HV_MAP_GPA_* here.

Reply:
That said, after reading your other comments, I now understand that hvdef and mshv_bindings have definition overlaps and were written by different teams. Sorry, I previously didn't fully understand your earlier point. I can find the equivalent of the "MSHV_SET_MEM_BIT_*" flags in hvdef and use those instead.

Reply:
I switched these flags to using the hvdef versions, which makes the code a lot cleaner.

Reply:
Copying John's offline comment in Teams support channel:

> I think I gave bad advice about the HV_MAP flags--I think those are inherently part of the mshv kernel interface and shouldn't be replaced with hvdef flags
>  
> so I think that part should be reverted, otherwise LGTM